snakePipes/shared/rscripts/CSAW_report.Rmd

```diff
@@ -72,7 +72,7 @@ Deeptools matrices were computed for the filtered lists of merged regions showin
 Heatmaps obtained using log2 ratio of ChIP signal over input were as follows:
 
 ```{r plot_l2r,fig.show='hold',fig.align='center', out.width ='40%',out.height='30%',fig.cap="UP and DOWN regions using log2 ratio."}
-if(sum(grepl("log2r.heatmap.png",snakemake@input[["csaw_in"]]))<1){
+if(sum(grepl("log2r.heatmap.png",snakemake@input))<1){
```

Comment:
`grepl` would coerce the `x` to a character vector, but I wonder how `as.character(snakemake@input)` would look like... I'm afraid it could return something weird (e.g. empty character vector) and then the `sum` will be always less than 1. Can we test this using some example where the if statement is expected to return True and confirm this? Also, adding a `print(as.character(snakemake@input))` before the if-statement (during this test) could be informative.

Reply:
I've tested this for ChIP-seq, for which I expect the sum to be >0, and it works (i.e. the Heatmap of log2r is included in the CSAW report). So I guess grepl coerced the `snakemake@input`, which is a named list, to a character vector. 
But, I do agree, it's cleaner to use the `[["csaw_in"]]` slot, which should be a character vector to start with. I'll have a look.